docs/src/ref/mcmc.md

```diff
@@ -414,6 +414,12 @@ end
 Note that when constructing involutions that call other `@involution` functions, the function being called (`bar` in this case) need not be mathematically speaking, an involution itself, for `foo` to be mathematically be an involution.
 Also, the top-level function must take three arguments (`model_args`, `proposal_args`, and `proposal_retval`), but any other `@involution` function may have an argument signature of the user's choosing.
 
+Some additional tips for defining valid involutions:
+
+- If you find yourself copying the same source address to multiple locations, it probably means your involution is not valid (the Jacobian matrix will have rows that are identical, and so the Jacobian determinant will be nonzero).
```

Comment:
Does this apply only to continuous choices? (The justification seems to.)

Reply:
True there, but even in the discrete case, the function cannot possible be an involution if we copy a discrete latent var to two addresses.  (Not sure whether we want to strictly enforce or let the user proceed at their own risk)

Reply:
@bzinberg Right, unless the second address is guaranteed to be the same as the first address, given the rest of the trace (which is possible in the case of a discrete choice).

Reply:
Yeah, you can have a valid involution that copies the discrete addresses that way. I'll specify that this tip only applies to continuous choices.